src/language/providers/attributeCompletion.ts

```diff
@@ -266,21 +274,28 @@ export function getAttributeCompletionProvider() {
   )
 }
 
-function getDefinedTypes(document: vscode.TextDocument) {
+function getDefinedTypes(
+  document: vscode.TextDocument,
+  position: vscode.Position
+) {
   var additionalTypes = ''
   var lineNum = 0
   const lineCount = document.lineCount
+  // const position = new vscode.Position(
+  //   lineCount,
+  //   document.lineAt(lineCount).range.end.character)
+  const nsPrefix = getNsPrefix(document, position)
```

Comment:
We call `getNSPrefix` quite a bit. Can it be called once per autocomplete instantiation and shared mong all the calls so we aren't constantly scanning for the schema tag? Or maybe we already do taht? It's not clear to me what the entrypoint into autocomplete is.